client/src/overview.jsx

```diff
@@ -77,17 +76,15 @@ class Overview extends Component {
       })
     );
 
-    const Container = collapsed ? OverviewCollapsed : OverviewContent;
-
     return (
-      <Container>
+      <OverviewContent collapsed={collapsed}>
         <div>
```

Comment:
Are these wrapper divs necessary? Wondering if you can handle multiple expressions without the div wrappers, like so: ```{mapper(rightContent)}
{mapper(leftContent)}```

Not sure if that's possible, but it'd be cool